### PR TITLE
Add Orlando file

### PR DESCRIPTION
**Pull request by Savanand**
This is the beginning of the Orlando file :sunny: 

Please provide feedback on

 - if you have never been to Orlando, what's bring you here
 - if you have been to Orlando, what thing you like most
 - are you interested in staying here?
 - @crichid what your child like to do here?
 - @beardofedu whats you drink choices here?

### CODE REVIEW

File: savanand-sanjose.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Orlando, FL
+
+### Restaurants
+
+### Local Attractions
+	Some places
```
Comment (diarmaidm) [inline savanand-sanjose.md]: @Savanand  I like that thinking :+1:

**Comment by diarmaidm** [comment]
> are you interested in staying here?

Orlando sounds good

**Comment by Savanand** [comment]
@diarmaidm yeah I think so :+1:

**Comment by readiu** [comment]
Ready to :ship:

**Comment by Savanand** [comment]
@readiu :+1:

**Comment by diarmaidm** [comment]
:+1: :shipit:

**Pull request merged**